packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -160,7 +160,7 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
           onDismiss={this.dismiss}
           className={_classNames.hiddenPanel}
         >
-          <div {...nativeProps} ref={this._panel} className={_classNames.root}>
```

Comment:
Is a nativeProps element allowed to override the new behavior?

Reply:
Having `aria-hidden` after `{...nativeProps}` means that any custom `aria-hidden` values (e.g: for some reason, if someone wanted the `Panel` to be `aria-hidden="true"` no matter what, it will be overridden by the `Panel`'s default behavior).  Is that what you were asking?  Or were you wanting it to be an option to override this default behavior?

Reply:
I was checking whether the decision to have the new behavior override any existing explicit `aria-hidden` settings was intentional or not; my expectation would have been that if a user is explicitly setting `aria-hidden`, the component should respect that (ie, for the new attribute on this line to appear before the `{...nativeProps}`). This doesn't impact me/my team specifically, I just wanted to confirm that you had intended this behavior since it could potentially impact other existing code unexpectedly.

Reply:
Gotcha.  Yeah, that makes sense.  I will get a few additional reviewers to get their opinion on this.

Reply:
I think if the user sets aria-hidden, that should override our default.